src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/response/HearingWithSessionsResponse.java

```diff
@@ -43,6 +45,7 @@ public HearingWithSessionsResponse(Hearing hearing) {
         this.reservedToJudge = hearing.getReservedJudge() != null ? hearing.getReservedJudge().getName() : null;
         this.sessions = hearing.getHearingParts()
             .stream()
+            .sorted(comparing(hearingPart -> hearingPart.getSession().getStart()))
             .map(h -> h.getSession() != null ? new ViewSessionResponse(h.getSession()) : null)
             .filter(Objects::nonNull)
```

Comment:
Can I suggest to do the filter as first thing,
then sorting
then mapping with ViewSessionResponse::new

that would reduce the null number of null checks

Reply:
sure, I don't know why I wrote it that way first time